README.md

```diff
@@ -115,4 +118,4 @@ The companies and organizations listed support the open, inclusive future provid
 | -------------|---------|
 | Argallab at Northwestern University | Founding Member |
 | Team Gleason | Founding Member |
-| LUCI | Founding Member, LUCI units with LuciCore X.X and newer software are compatible with USB and BLE WDI controllers. | 
+| [LUCI](https://www.luci.com) | Founding Member, LUCI units with LuciCore X.X and newer software are compatible with USB and BLE WDI controllers. |
```

Comment:
Update to "LuciCore 2.0 and newer..." pls. 
Question: Should we pull the BLE compatibility statement out?

Reply:
Yes I think that will be implemented later. We would need an interface for pairing BLE devices, which I doubt we'll get in time for this.